app/api/documents/specs/deprecatedRoutes.spec.js

```diff
@@ -12,6 +12,10 @@ jest.mock('../../utils/languageMiddleware.ts', () => (req, _res, next) => {
   next();
 });
 
+jest.mock('../../auth/headersMiddleware.ts', () => (_req, _res, next) => {
+  next();
+});
+
```

Comment:
why is this mock needed ? i think now the middleware is not being used on the test environment right ?

Reply:
Yes it is not used. Removed